src/main/java/mate/controller/cars/CarDriversController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller.cars;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers")
+public class CarDriversController extends HttpServlet {
```

Comment:
we should start controller’s name with a verb
```suggestion
public class GetCarDriversController extends HttpServlet {
```